BrowserKit/Sources/ToolbarKit/AddressToolbar/LocationView/LocationView.swift

```diff
@@ -171,7 +171,7 @@ final class LocationView: UIView, LocationTextFieldDelegate, ThemeApplicable, Ac
 
             iconContainerBackgroundView.topAnchor.constraint(equalTo: urlTextField.topAnchor),
             iconContainerBackgroundView.bottomAnchor.constraint(equalTo: urlTextField.bottomAnchor),
-            iconContainerBackgroundView.leadingAnchor.constraint(lessThanOrEqualTo: urlTextField.leadingAnchor),
+            iconContainerBackgroundView.leadingAnchor.constraint(equalTo: iconContainerStackView.leadingAnchor),
```

Comment:
Nice catch @PARAIPAN9! This does look a bit unexpected. I think I have to defer to @thatswinnie on this one—not sure if there was a good reason not to use `equalTo` here? 🤔  But upon quick inspection, I don't see the harm of this, and it may help the issue.

Could either of you also explain the purpose of the `iconContainerBackgroundView`? 🙏  Seems I can remove it completely and I don't notice any visual changes? Which makes me wonder if this constraint could even be the culprit in the first place... it's a mysterious problem! 🤔

Reply:
lessThanOrEqual was a modification of mine to my other PR that may caused this problem and the backgroundView is there to hide the text behind the lock when dynamic text is used.

Reply:
Thanks @PARAIPAN9 for the background info! 😁 🙏